scripts/release/package_audit.rb

```diff
@@ -110,16 +110,16 @@
 end
 
 puts "\n\nRun the following:"
-puts "cd openc3-cosmos-init/plugins; yarn install; yarn upgrade-interactive --latest; cd ../.."
-puts "cd playwright; yarn install; yarn upgrade-interactive --latest; cd .."
-puts "cd docs.openc3.com; yarn install; yarn upgrade-interactive --latest; cd .."
+puts "cd openc3-cosmos-init/plugins; pnpm install; pnpm update --interactive --latest; cd ../.."
+puts "cd playwright; pnpm install; pnpm update --interactive --latest; cd .."
+puts "cd docs.openc3.com; pnpm install; pnpm update --interactive --latest; cd .."
```

Comment:
It's literally the exact same command and params: `update --interactive --latest`. Nice.